editor/src/components/inspector/resize-to-fit-control.spec.browser2.tsx

```diff
@@ -49,6 +49,44 @@ describe('Resize to fit control', () => {
     expect(view.style.flexGrow).toEqual('')
     expect(view.style.flexBasis).toEqual('')
   })
+
+  it('when container is already set to hug, it is converted back to fixed size', async () => {
+    const editor = await renderTestEditorWithCode(project, 'await-first-dom-report')
+    const view = await selectView(editor)
+    await clickResizeToFit(editor)
+    await expectSingleUndoStep(editor, () => clickResizeToFit(editor))
+
+    expect(view.style.width).toEqual('221px')
+    expect(view.style.minWidth).toEqual('')
+    expect(view.style.maxWidth).toEqual('')
+    expect(view.style.height).toEqual('154px')
+    expect(view.style.minHeight).toEqual('')
+    expect(view.style.maxHeight).toEqual('')
+    expect(view.style.flex).toEqual('')
+    expect(view.style.flexShrink).toEqual('')
+    expect(view.style.flexGrow).toEqual('')
+    expect(view.style.flexBasis).toEqual('')
+  })
+
+  it('when container is already set to hug, it is converted back to fixed size, with shortcut', async () => {
+    const editor = await renderTestEditorWithCode(project, 'await-first-dom-report')
+    const view = await selectView(editor)
+    await clickResizeToFit(editor)
+    await expectSingleUndoStep(editor, async () =>
+      pressKey('r', { modifiers: { alt: true, cmd: true, shift: true, ctrl: false } }),
+    )
+
+    expect(view.style.width).toEqual('221px')
+    expect(view.style.minWidth).toEqual('')
+    expect(view.style.maxWidth).toEqual('')
+    expect(view.style.height).toEqual('154px')
+    expect(view.style.minHeight).toEqual('')
+    expect(view.style.maxHeight).toEqual('')
+    expect(view.style.flex).toEqual('')
+    expect(view.style.flexShrink).toEqual('')
+    expect(view.style.flexGrow).toEqual('')
+    expect(view.style.flexBasis).toEqual('')
+  })
```

Comment:
Can you add a test for the case where one axis is set to hug and the other set to fixed please

Reply:
added in https://github.com/concrete-utopia/utopia/pull/3215/commits/07724d722dcb0054f281e807e757008095ca61a2